settings.gradle

```diff
@@ -41,3 +37,8 @@ boolean omnitureDependenciesAreSatisfied() {
     File dep = file("${System.getProperty("user.home")}/libs/adobeMobileLibrary.jar")
     return dep.exists()
 }
+
+
+// For building exoplayer-specific sample apps, which are located under the 'exoplayer' directory.
```

Comment:
This section looks fishy to me.  Perhaps an in-person demo would make it clear to me what the intent is here.